Configuration/StandardSequences/python/AlCaRecoStreams_cff.py

```diff
@@ -17,12 +18,12 @@
 from Alignment.CommonAlignmentProducer.ALCARECOTkAlCosmics0THLT_cff import *
 # AlCaReco for track based alignment using isoMu events
 from Alignment.CommonAlignmentProducer.ALCARECOTkAlMuonIsolated_cff import *
-# AlCaReco for track based alignment using isoMu events for PA data-taking
```

Comment:
It was to put the three ALCARECOs under the same umbrella of isoMu events.  I'll go back and put in comments for the PA producers as well as the HI ones.